File: python_ruby/OOP_variables/4.rb
class C
  attr_reader :value
  attr_writer :value
  attr_accessor :value
  def initialize(v)
    @value = v
  end
  def show()
    p @vlue
  end
  def getValue()
    return @value
  end
  def setValue(v)
    @value = v
  end
end

c1 = C.new(10)
p c1.value
c1.value = 20
p c1.value

File: python_ruby/OOP_variables/4.py
class C(object):
    def __init__(self, v):
        self.__value = v
    def show(self):
        print(self.__value)
    def getValue(self):
        return self.value
    def setValue(self, v):
        self.value = v

c1 = C(10)

c1.show()
